.github/workflows/build-node-examples.yml

```diff
@@ -0,0 +1,110 @@
+name: build node examples from source
+
+on: pull_request
+
+jobs:
+  node:
+    runs-on: ${{ matrix.platform }}
+    timeout-minutes: 40
+
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ubuntu-latest, macos-latest, windows-latest]
+        example:
+          - name: GatsbyThemedSite
+            folder: react/gatsby-themed-site
+            executable: gatsby-themed-site-app
+          - name: CRAApp
+            folder: react/create-react-app
+            executable: app
+          - name: NextjsApp
+            folder: react/next.js
+            executable: app
+          - name: VanillajsMonolithApp
+            folder: vanillajs/monolith
+            executable: app
+          - name: quasar-app
+            folder: vue/quasar-app
+            executable: app
+          - name: svelte-app
+            folder: svelte/svelte-app
+            executable: app
+        include:
+          - platform: ubuntu-latest
+            CARGO_HOME: ~/.cargo
+            releaseFolder: target/release/bundle/deb
+            ext: _0.1.0_amd64.deb
+          - platform: macos-latest
+            CARGO_HOME: ~/.cargo
+            releaseFolder: target/release/bundle/osx
+            ext: .app
+          - platform: windows-latest
+            CARGO_HOME: ~/.cargo
+            releaseFolder: target/release
+            ext: .x86.msi
+
+    steps:
+      - name: checkout tauri
+        uses: actions/checkout@v2
+        with:
+          repository: tauri-apps/tauri
+          ref: dev
+      - name: checkout examples
+        uses: actions/checkout@v2
+        with:
+          path: examples
```

Comment:
maybe this is one examples too many

Reply:
The idea with this is to emulate what we had before. A `tauri` folder with `tauri/examples`. I _think_ this is correct.